doc/help/pypeit_install_telluric.rst

```diff
@@ -9,5 +9,9 @@
       -h, --help   show this help message and exit
       --path PATH  Path to directory with TelFit files downloaded from the PypeIt
                    Google Drive Telluric/ folder (default:
+<<<<<<<<< Temporary merge branch 1
```

Comment:
@suksien -- there are more examples to fix

Reply:
fixed to be same as path on develop (probably ok?)